### ISSUE TITLE
wiktionary module should redirect

### ISSUE THREAD

**Issue by jonorthwash** [goavki/phenny#239]
Current behaviour:
```
(20:41:40) firespeaker: .w Joke#English
(20:41:42) begiak: Perhaps you meant 'joke'?
```

Instead, it should redirect to the word suggested and return the definition there.

**Comment by vikramkashyap** [comment]
I'll give this a try. I think it should still tell the user that it redirected, though.

**Comment by shardulc** [comment]
Fixed via 413ffd0.
